src/join-9.sol

```diff
@@ -56,7 +56,11 @@ contract GemJoin9 {
         vat = VatLike(vat_);
         ilk = ilk_;
         gem = GemLike(gem_);
-        dec = GemLike(gem_).decimals();
+
+        uint256 dec_ = GemLike(gem_).decimals();
+        require(dec_ <= 18, "GemJoin9/decimals-19-or-higher");
```

Comment:
This actually should be only 18 decimals, or otherwise we should account for the decimals before calling `slip` (as done in the other gemJoins)

Reply:
Done in addf9cd

Reply:
I see your point but I think adding this `feat` should be discussed broadly and see if there is consensus in adding it, considering the ongoing audit and the proxy actions changes linked above.

Reply:
Ok, let's have a short discussion next week (it's ok, we many times go over the changes and responses).